File: C++/class and functions.cpp
#include<bits/stdc++.h>
using namespace std;

class Car{
private:
    float price;
    char name[50];
public://member function
    void getData();//declaration
    void showData();
};

//definition
void Car::getData(){
    cin>>price>>name;
}

void Car::showData(){
    cout<<name<<" "<<price;
}

int main(){
    Car car;
    car.getData();
    car.showData();
    return 0;
}

File: Python/class and functions.py
class Car:
	pass	

def getData(self):
	#multiple input
	self.name, self.model, self.price = map(str,input("name, model, price ?:").split())
	# self.name = input()
	# self.model = input()
	# self.price = input()

def  showData(self):
	print(self.name + " " + self.model + " " + self.price)






if __name__ == "__main__":
	#member function of the Car class
	Car.getData = getData 
	Car.showData = showData

	car = Car()
	car.getData()
	car.showData()
